layouts/index.hbs

```diff
@@ -73,7 +73,7 @@
           </div>
         {{/if}}
         <p>
-          {{{ labels.version-schedule-prompt }}} <a href="https://github.com/nodejs/LTS#lts-schedule1">{{ labels.version-schedule-prompt-link-text }}.</a>
+          {{{ labels.version-schedule-prompt }}} <a href="https://github.com/nodejs/LTS#nodejs-release-working-group">{{ labels.version-schedule-prompt-link-text }}.</a>
```

Comment:
@Webysther Thank for doing this 🥇 
I believe the new URL should be https://github.com/nodejs/Release#release-schedule1

Reply:
@refack Oh sorry! That's true, fixed!